mobile/src/components/chat-conversation/components/message-form/message-form.tsx

```diff
@@ -24,14 +23,15 @@ const MessageForm: FC<Props> = ({ onSubmit }) => {
   const { control, errors, handleSubmit, reset, watch } =
     useAppForm<ChatMessageFormRequestDto>({
       defaultValues: getDefaultMessagePayload(),
-      validationSchema: chatMessageCreate,
     });
 
   const [messageChar, setMessageChar] = useState<string>();
   const [rowsCount, setRowsCount] = useState<number>(1);
+  const [isDisabledButton, setIsDisabledButton] = useState<boolean>(true);
 
   const hitSlop = { top: 5, bottom: 5, left: 5, right: 5 };
-  const hasError = Boolean(errors.message?.message);
+  const hasError = Boolean(errors.message?.message) || isDisabledButton;
+  const watchMessage = watch('message');
```

Comment:
let's do this simpler, without useState and useEffect
```suggestion
  const hasError = Boolean(errors.message?.message);
  const messageFieldValue = watch('message');
  const isButtonDisabled = hasError || Boolean(messageFieldValue.trim());
```

Reply:
But if we havn't validation maybe just `const isButtonDisabled = Boolean(!messageFieldValue.trim());` ?

Reply:
it better to use it in this case, because if  we change the way how we control validation or will add it in future, we won't need to think about it

Reply:
input shouldn't know how errors are handled that's why we should leave this